packages/language-picker/src/language-picker.tsx

```diff
@@ -208,7 +212,11 @@ function LanguagePicker< TLanguage extends Language >( {
 				<div className="language-picker-component__language-buttons">
 					{ languagesToRender.map( ( language ) => (
 						<Button
-							isPrimary={ selectedLanguage && language.langSlug === selectedLanguage.langSlug }
+							variant={
```

Comment:
The styling seems to be off now, tested in calypso.live.

Before:
<img width="1602" alt="language-picker-before" src="https://github.com/Automattic/wp-calypso/assets/7847633/0cf2a3d9-f952-4d70-8b17-2fa449abbf39">

After:

<img width="1602" alt="language-picker-after" src="https://github.com/Automattic/wp-calypso/assets/7847633/664c0fff-d82a-40bf-90b2-3c0026637fda">

Reply:
Thanks! Turns out the issue is that I assumed the `secondary` variant of the button was the default, but it's not. I restored the previous styles by changing variant to undefined when it's not primary.